src/tools.ts

```diff
@@ -210,12 +248,27 @@ async function _callTool(
 export async function loadMcpTools(
   serverName: string,
   client: Client,
-  throwOnLoadError: boolean = true
+  options?: LoadMcpToolsOptions
 ): Promise<StructuredToolInterface[]> {
+  const {
+    throwOnLoadError,
+    prefixToolNameWithServerName,
+    additionalToolNamePrefix,
+  } = {
+    ...defaultLoadMcpToolsOptions,
+    ...(options ?? {}),
+  };
+
   // Get tools in a single operation
   const toolsResponse = await client.listTools();
   getDebugLog()(`INFO: Found ${toolsResponse.tools?.length || 0} MCP tools`);
 
+  const initialPrefix = additionalToolNamePrefix
```

Comment:
ha! there was this discussion briefly in py https://github.com/langchain-ai/langchain-mcp-adapters/issues/18

it's probably good to add this as an option